tasks/service-runner/app/workers/pull-worker.ts

```diff
@@ -150,7 +170,7 @@ async function _pullAndDoWork(repeat = true): Promise<void> {
             }
           }
           if (!complete) {
-            if (tries < maxRetries) {
+            if (tries < maxItemUpdateRetries) {
               logger.info(`Retrying failure to update work item with id ${workItem.id} for job id ${workItem.jobID}`);
               await sleep(1000);
```

Comment:
Shouldn't we rely on the `axiosRetry` exponential backoff here instead of using `sleep`?

Reply:
Same comment as above https://github.com/nasa/harmony/pull/163/files#r829200989 but if you think it's essential for this ticket I can add that.

Reply:
Or I can also file a new ticket. Up to you.

Reply:
Actually axisoRetry wraps every call, so I should probably refactor this regardless.